apps/dashboard/config/application.rb

```diff
@@ -38,7 +38,7 @@ class Application < Rails::Application
     config.exceptions_app = self.routes
 
     if ::Configuration.load_external_config?
-      config.paths["config/initializers"] << ::Configuration.config_root.join("initializers").to_s
+      config.paths["config/initializers"].unshift ::Configuration.config_root.join("initializers").to_s
       config.paths["app/views"].unshift ::Configuration.config_root.join("views").to_s
```

Comment:
Can we add a comment here saying that we _want_ the things in `/etc/` to go first? or that that's somehow purposeful placing them first.

Reply:
Np, I actually had a comment originally then removed it 🙃.